addons/auth_signup/controllers/main.py

```diff
@@ -60,7 +60,15 @@ def web_auth_signup(self, *args, **kw):
                 )
                 template = request.env.ref('auth_signup.mail_template_user_signup_account_created', raise_if_not_found=False)
                 if user_sudo and template:
-                    template.sudo().send_mail(user_sudo.id, force_send=True)
+                    template.sudo().with_context(
+                        email_notification_force_header=True,
+                        email_notification_force_footer=True,
+                        email_notification_subtitles=[_("Your Account"), user_sudo.name or ''],
```

Comment:
Thinking about it, this leads to lang incoherency ... it is going to use current user lang, while the template can be dynamically translated into recipient's lang.

Previously the whole template was translated, hence the subtitles were translated. It won't be the case anymore, will it ? Somehow we have to find a solution for that ... delaying the translation of subtitles to the rendering phase, when we know the lang.

Reply:
Good catch, indeed. As discussed, I have used LazyTranslation to solve that problem.